### DISCUSSION TITLE
No community meeting in September so we can attend an IMPORTANT seminar?

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#185] [Meetings]
Hi all,

I propose we postpone our September meeting in favour of attending this important seminar - 

https://www.helmholtz-hirse.de/series/2024_09_23-seminar_32.html

> Sustainability Prize to ESMValTool: The Earth System Model Evaluation Tool (ESMValTool) is an open-source community-developed diagnostics and performance metrics tool for the evaluation and analysis of Earth System Models (ESMs). ESMValTool allows for a comparison of single or multiple models against predecessor versions and observations.
Speaker: Birgit Hassler (DLR)

Thoughts?

**Comment by bouweandela** [comment]
Congratulations to all of us for winning this prize! :1st_place_medal: 

It would be nice to have a community meeting again soon though, are you planning to organize one in October @paultgriffiths?

**Comment by rswamina** [comment]
@paultgriffiths - I would like to propose the following two agenda items for the next Community meeting:

1. Discussion on ESMValTool governance:
Our current governance document is at :https://esmvaltool.org/team/
Some ideas for re-drafting  our document is at: https://sustainers.github.io/governance-readiness/
It would be good to have a discussion on how to write our governance structure. Bouwe is best placed to lead this topic, so please tag him.

2. Discussion on preparations for ESGF phase out of CMIP5 and CMIP3 data:
This is described in a discussion and it will be good for the community to provide input here: https://github.com/ESMValGroup/Community/discussions/181
